### ISSUE TITLE
All blocks that require a pickaxe to break can no longer be harvested in survival

### ISSUE THREAD

**Issue by notWaterGuy12** [Layers-of-Railways/Railway#137]
Mod Version: 1.3.2
Forge Version: 43.2.14
Minecraft Version: 1.19.2

No idea how this happened but a pretty funny bug tbh

**Comment by muriplz** [comment]
- 1.18: https://github.com/Layers-of-Railways/Railway/pull/135 (pickaxe fix)
- 1.19: https://github.com/Layers-of-Railways/Railway/pull/136 (pickaxe + track recipe fixes)

**Comment by SomeBody16** [comment]
```java
[15Jun2023 11:45:47.822] [Worker-Main-48/ERROR] [net.minecraft.tags.TagLoader/]: Couldn't load tag minecraft:mineable/pickaxe as it is missing following references: railways:track_hexcasting_edified (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_aspen (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_baobab (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_blue_enchanted (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_bulbis (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_cherry (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_cika (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_cypress (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_ebony (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_embur (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_ether (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_fir (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_green_enchanted (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_holly (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_imparius (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_jacaranda (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_lament (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_mahogany (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_maple (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_nightshade (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_palm (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_pine (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_rainbow_eucalyptus (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_redwood (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_skyris (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_sythian (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_white_mangrove (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_willow (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_witch_hazel (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_byg_zelkova (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_blue_skies_bluebright (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_blue_skies_cherry (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_blue_skies_dusk (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_blue_skies_frostbright (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_blue_skies_lunar (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_blue_skies_maple (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_blue_skies_starlit (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_twilightforest_canopy (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_twilightforest_darkwood (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_twilightforest_mangrove (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_twilightforest_minewood (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_twilightforest_sortingwood (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_twilightforest_timewood (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_twilightforest_transwood (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_twilightforest_twilight_oak (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_biomesoplenty_cherry (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_biomesoplenty_dead (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_biomesoplenty_fir (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_biomesoplenty_hellbark (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_biomesoplenty_jacaranda (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_biomesoplenty_magic (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_biomesoplenty_mahogany (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_biomesoplenty_palm (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_biomesoplenty_redwood (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_biomesoplenty_umbran (from Steam_Rails-1.3.1+forge-mc1.18.2.jar), 
	railways:track_biomesoplenty_willow (from Steam_Rails-1.3.1+forge-mc1.18.2.jar)
```

**Comment by IThundxr** [comment]
fixed in 1.3.4
